database/mssql/scripts/versions/v_7_ddl.sql

```diff
@@ -0,0 +1,66 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+CREATE TABLE [permit].[ORBC_TRANSACTION](
+	[TRANSACTION_ID] [bigint] IDENTITY(1,1) NOT NULL,
+	[MERCHANT_ID] [int] NOT NULL,
+	[TRANSACTION_TYPE] [varchar] (3) NOT NULL,
+	[TRANSACTION_ORDER_NUMBER] [varchar](30) NOT NULL,
+	[TRANSACTION_AMOUNT] [decimal] (9, 2) NOT NULL,
+	[TRANSACTION_APPROVED] [tinyint] NOT NULL CHECK (TRANSACTION_APPROVED BETWEEN 0 AND 1),
+	[AUTH_CODE] [varchar] (32) NOT NULL,
+	[TRANSACTION_CARD_TYPE] [nvarchar](2) NOT NULL,
+	[TRANSACTION_DATE] [datetime2](7) NULL,
+	[CVD_ID] [tinyint] NOT NULL CHECK (CVD_ID BETWEEN 1 AND 6),
+	[PAYMENT_METHOD] [nvarchar](2) NOT NULL,
+	[MESSAGE_ID] [int] NOT NULL,
+	[MESSAGE_TEXT] [varchar](100) NOT NULL,
+	[CONCURRENCY_CONTROL_NUMBER] [int] NULL,
+	[DB_CREATE_USERID] [varchar](63) NOT NULL,
+	[DB_CREATE_TIMESTAMP] [datetime2](7) NOT NULL,
+	[DB_LAST_UPDATE_USERID] [varchar](63) NOT NULL,
+	[DB_LAST_UPDATE_TIMESTAMP] [datetime2](7) NOT NULL,
+ CONSTRAINT [PK_ORBC_TRANSACTION] PRIMARY KEY CLUSTERED 
+(
+	[TRANSACTION_ID] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+CREATE TABLE [permit].[ORBC_PERMIT_TRANSACTION](
+	[ID] [bigint] IDENTITY(1,1) NOT NULL,
+	[PERMIT_ID] [bigint] NOT NULL,
+	[TRANSACTION_ID] [bigint] NOT NULL,
+	[CONCURRENCY_CONTROL_NUMBER] [int] NULL,
+	[DB_CREATE_USERID] [varchar](63) NOT NULL,
+	[DB_CREATE_TIMESTAMP] [datetime2](7) NOT NULL,
+	[DB_LAST_UPDATE_USERID] [varchar](63) NOT NULL,
+	[DB_LAST_UPDATE_TIMESTAMP] [datetime2](7) NOT NULL,
+ CONSTRAINT [PK_ORBC_PAYMENT_TRANSACTIONS] PRIMARY KEY CLUSTERED 
+(
+	[ID] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+ALTER TABLE [permit].[ORBC_PERMIT_TRANSACTION]  WITH CHECK ADD  CONSTRAINT [FK_ORBC_PERMIT_TRANSACTION_PERMIT_ID] FOREIGN KEY([PERMIT_ID])
+REFERENCES [permit].[ORBC_PERMIT] ([ID])
+GO
+
+ALTER TABLE [permit].[ORBC_PERMIT_TRANSACTION]  WITH CHECK ADD  CONSTRAINT [FK_ORBC_PERMIT_TRANSACTION_TRANSACTION_ID] FOREIGN KEY([TRANSACTION_ID])
+REFERENCES [permit].[ORBC_TRANSACTION] ([TRANSACTION_ID])
+GO
+
+
+DECLARE @VersionDescription VARCHAR(255)
+SET @VersionDescription = 'Initial creation of entities for Permit Payment'
+
+INSERT [dbo].[ORBC_SYS_VERSION] ([VERSION_ID], [DESCRIPTION], [DDL_FILE_SHA1], [RELEASE_DATE]) VALUES (7, @VersionDescription, '$(FILE_HASH)', getdate())
+
+
+COMMIT
```

Comment:
This needs a corresponding BEGIN TRANSACTION at the top

Reply:
Yeah, I will correct it. Thanks.